pylabrobot/liquid_handling/backends/hamilton/STAR.py

```diff
@@ -385,10 +406,19 @@ def __init__(self, num_channels: int = 8, read_timeout: int = 5, **kwargs):
     self.dev: Optional[usb.core.Device] = None
     self._tip_types: dict[str, int] = {}
     self.num_channels = num_channels
+    self._iswap_parked: bool = False
 
     self.read_endpoint: Optional[usb.core.Endpoint] = None
     self.write_endpoint: Optional[usb.core.Endpoint] = None
 
+  @property
+  def iswap_parked(self):
+    return self._iswap_parked
+
+  @iswap_parked.setter
+  def iswap_parked(self, parked: bool):
+    self._iswap_parked = parked
```

Comment:
I don't really see value in having a setter over just writing to `_iswap_parked` directly, since we only write from within self. In fact, this gives the impression that the setter is publicly available, which is not the case (the state should only be handled indirectly, by STAR itself).